build.sbt

```diff
@@ -85,16 +140,27 @@ lazy val runtime = project
   .settings(
     sharedSettings,
     moduleName := "mdoc-runtime",
-    libraryDependencies ++= List(
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided,
-      "com.lihaoyi" %% "pprint" % pprintVersion.value
+    unmanagedSourceDirectories.in(Compile) := multiScalaDirectories("runtime").value,
+    libraryDependencies ++= crossSetting(
+      scalaVersion.value,
+      if3 = List(
+        // "ch.epfl.lamp" %% "dotty-reflect" % scalaVersion.value, <- not yet sure what will be needed for Scala 3
+        // "com.lihaoyi" %% "sourcecode" % "0.2.1",
+        ("com.lihaoyi" %% "pprint" % pprintVersion.value)
```

Comment:
This will be ported soon, but it's still being worked on so there is a number of places that have pprint replaced for now.